packages/@azure/servicebus/data-plane/test/batchReceiver.spec.ts

```diff
@@ -19,6 +19,8 @@ import {
 
 import {
   testSimpleMessages,
+  testMessagesToSamePartitions,
+  testMessagesToSamePartitionsWithSessions,
```

Comment:
Do we need `testSimpleMessages` anymore?
Why not use `testMessagesToSamePartitions` everywhere instead?
And when doing so, lets remove the "ToSamePartitions` from the name.
It will be misleading when we use it for non-partitions

Reply:
I was using `testSimpleMessages` for single message and ``testMessagesToSamePartitions`` for multiple message.
I'll remove `testSimpleMessages` and use `testMessages` everywhere.